android/res/layout/item_phone.xml

```diff
@@ -0,0 +1,47 @@
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/height_item_oneline"
+    android:background="?clickableBackground"
+    android:gravity="center_vertical"
+    android:orientation="horizontal"
+    android:paddingEnd="@dimen/margin_base"
+    android:paddingLeft="@dimen/margin_half_plus"
+    android:paddingRight="@dimen/margin_base"
```

Comment:
Redundant tags "android:paddingLeft" and "android:paddingRight".  Min api level is 21.